shap/utils/_transformers.py

```diff
@@ -0,0 +1,130 @@
+import torch
+import numpy as np
+import scipy as sp
+
+class GenerateLogits:
```

Comment:
nit: wondering if there is a better class name for this, since it seems to do more than this, it already has a generate_logits method and other methods as well?

Reply:
hmm maybe LogitTransformer or LogitGenerator, not really sure what might be better

Reply:
maybe LogitGeneratorModel might be more descriptive?